### PR TITLE
Remove diskPath from DiskGet

### PR DESCRIPTION
**Pull request by derekbit**

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -29,3 +30,10 @@ func GetVolumeNameFromEngineName(engineName string) string {
 func BytesToMiB(bytes uint64) uint64 {
 	return bytes / 1024 / 1024
 }
+
+func RemovePrefix(path, prefix string) string {
+	if strings.HasPrefix(path, prefix) {
+		return strings.TrimPrefix(path, prefix)
+	}
+	return path
```
Comment (innobead) [inline pkg/util/util.go:38]: Can just `return strings.TrimPrefix(path, prefix)`

**Review by innobead — APPROVED** [review]
LGTM.

File: pkg/util/util.go (hunk 1 above)
Comment (shuo-wu) [reply pkg/util/util.go:38]: NIT: is this function necessary...? Why not directly call `strings.HasPrefix`...?

Comment (derekbit) [reply pkg/util/util.go:38]: Not necessary. Can do `diskPath := strings.TrimPrefix(path, prefix)` directly

**Pull request merged**